src/store.js

```diff
@@ -44,7 +44,18 @@ export const store = reactive({
     template: '',
     include_test: false,
     output_dir: './logs',
-    log_every_iters: 10
+    log_every_iters: 10,
```

Comment:
Can we create a single file from where we can import the default config dict instead of coding it in 3 places ?

Reply:
Yes, we can, working on it :+1: